docs/playroom/snippets.ts

```diff
@@ -344,6 +346,51 @@ const snippets = [
 			</Text>
 		</Footer></Box>`,
 	},
+	{
+		group: 'Drawer',
```

Comment:
When you preview or add these snippets, the focus is continually cycled between all the titles making playroom unusable…

Reply:
Removed the snippets since they are too problematic